documents/chunk_size.md

```diff
@@ -46,4 +45,18 @@ Here are the questions:
 > - What is Huss an example of?
 > - How did the author feel when she took an exam?
 > - What is one lesson that the author learned through attending college?
-> - What should the author do to improve her life?
\ No newline at end of file
+> - What should the author do to improve her life?
+
+## Conclusion
+**Quarter a paragraph** was the best chunk size among the four options in our evaluation method.
+
+A sentence was too short for LLM to understand the context. It gave several unrelated wrong answers.
+Half a paragraph was too long, so a similar chunk to the question was incorrectly picked.
+A paragraph was too long, it distracted LLM from the main point of the question and answer.
+
+Quarter a paragraph also gave answers mixed with incorrect information, but this one had fewer incorrect answers and more correct answers.
+A quarter of a paragraph includes 1-4 sentences each.
+
+Here is an insight from reading the responses:
+The degree of personalization in response depends greatly on the prompt you give to the LLM model. Some gave general answers that were not incorrect but not aligned with the author's writing.
+We may need to specify in the prompt to what degree we want the response to be personalized.
```

Comment:
Yes, I see that many AI engineers out there work on prompt engineering to improve output accuracy

Reply:
I think I will also need to look into prompt engineering at some point then! Thank you for the information